starters/deno-oak-denodb/src/main.ts

```diff
@@ -12,7 +13,7 @@ router.get('/', ({ request, response }: Context) => {
 app.use(
   oakCors({
     allowedHeaders: ['Content-Type', 'Authorization'],
-    origin: CORS_ALLOWED_ORIGINS ? CORS_ALLOWED_ORIGINS.split(',') : '',
+    origin: corsAllowedOrigins(),
```

Comment:
I extracted this to a separate method just so that I can write JSDoc on it and verify the documentation is properly generated.